sys/dev/puc/puc_pci.c

```diff
@@ -198,3 +198,5 @@ static driver_t puc_pci_driver = {
 };
 
 DRIVER_MODULE(puc, pci, puc_pci_driver, puc_devclass, 0, 0);
+MODULE_PNP_INFO("U16:vendor;U16:device;D:#", pci, puc, puc_pci_devices,
```

Comment:
This descriptor string is wrong in exactly the same way as the one for `bce(4)`.  You need to actually look at the struct definition for the devices array to determine the correct format string.

For this driver, the device table data itself will need some modification because the table assumes that a subvendor ANY match means the subdevice id is ignored, and so 0 is often used for table subdevice id when subvendor is ANY.  I think many subdevice id zeros in this driver need to be converted to 0xffff.